ReactiveXComponent/ReactiveXComponent/communication/publisher.py

```diff
@@ -0,0 +1,134 @@
+import websocket
+from ReactiveXComponent.configuration.serializer import *
+import json
+import ssl
+import time
+from threading import Thread as thread
+from lxml import etree
+"""Module test_send"""
+            
+class publisher:
+	
+	def getXmlContent(self):
+		tree=etree.parse(self.file)
+		tree1=etree.tostring(tree)
+		rt=etree.fromstring(tree1)
+		n={'xmlns':'http://xcomponent.com/DeploymentConfig.xsd'}
+		self.root=rt
+		self.ns=n
+	
+	def findComponent(self,componentName):
```

Comment:
follow convention: https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables

method names = all lower case words separated by '_'